ansible/files/prosody.cfg.lua

```diff
@@ -193,6 +193,14 @@ else
 	statistics_interval = 60
 end
 
+if ENV_SNIKKET_TWEAK_DNSSEC == "1" then
+	local trustfile = "/usr/share/dns/root.ds"; -- Requires apt:dns-root-data
+	-- Bail out if it doesn't work
+	assert(require"lunbound".new{ trustfile = trustfile }:resolve ".".secure,
```

Comment:
Here it needs to pass `resolvconf = true` in the table to behave the same as when Prosody initializes it.